### PR TITLE
Adding Threat Intelligence Workbook as a part of the solution

### PR DESCRIPTION
**Pull request by samikroy**
Adding Threat Intelligence Workbook as a part of the solution

### CODE REVIEW

**Review by v-prasadboke — APPROVED** [review]

**Comment by v-prasadboke** [comment]
Hello @samikroy all looks good, workbook is deployed successfully.
Just resolve the branch conflict to merge the PR

**Review by v-prasadboke — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

File: Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json [hunk 1]
```diff
@@ -4979,6 +4979,19 @@
   "subtitle": "",
   "provider": "Microsoft"
 },
+{
+    "workbookKey": "ThreatIntelligenceWorkbook",
```
Comment (v-sabiraj) [inline Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json:4983]: @samikroy, this "workbookKey" is already used for the threat intelligence workbook in [threat intelligence](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Threat%20Intelligence/Workbooks) solution. Can you please change this?

File: Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json [hunk 2]
```diff
@@ -4979,6 +4979,19 @@
...
+    "logoFileName": "",
+    "description": "Gain insights into threat indicators ingestion and search for indicators at scale across Microsoft 1st Party, 3rd Party, On-Premises, Hybrid, and Multi-Cloud Workloads. Indicators Search facilitates a simple interface for finding IP, File, Hash, Sender and more across your data. Seamless pivots to correlate indicators with Microsoft Sentinel: Incidents to make your threat intelligence actionable.",
+    "dataTypesDependencies": [ "ThreatIntelligenceIndicator", "SecurityIncident" ],
+    "dataConnectorsDependencies": [ "ThreatIntelligence", "ThreatIntelligenceTaxii" ],
+    "previewImagesFileNames": [ "ThreatIntelligenceWhite.png", "ThreatIntelligenceBlack.png" ],
+    "version": "1.0.0",
+    "title": "Threat Intelligence",
+    "templateRelativePath": "ThreatIntelligence.json",
```
Comment (v-sabiraj) [inline Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json:4991]: Also please change the "templateRelativePath" and workbook file name as that is also same as [here](https://github.com/Azure/Azure-Sentinel/tree/master/Solutions/Threat%20Intelligence/Workbooks).

**Comment by v-sabiraj** [comment]
@samikroy, please raise a new pr for these changes, else please suggest what should we change with, thanks.

File: Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json (hunk 1 above)
Comment (samikroy) [reply Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json:4983]: Addressed in PR #7776

File: Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json (hunk 2 above)
Comment (samikroy) [reply Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json:4991]: Addressed in PR #7776

**Pull request merged**